portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,6 +33,9 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+//for user API
```

Comment:
Feel free to leave out comments for import lines – often times there are tools for automating removal of unnecessary imports and addition of required ones so you shouldn't have to worry too much documenting these.

Good intention here though!